sdk/core/azure-core-amqp/inc/azure/core/amqp/models/amqp_value.hpp

```diff
@@ -1070,14 +1061,15 @@ namespace Azure { namespace Core { namespace Amqp { namespace Models {
      * @returns The descriptor for this composite type.
      */
     AmqpValue const& GetDescriptor() const { return m_descriptor; }
-
+#if 0
```

Comment:
Should we drop it entirely, or you still may need it in the future?

Reply:
I need to do a deeper cleanup pass on the change - there's a bunch of debugging artifacts there, including this one, I suspect.